### DISCUSSION TITLE
a question to help me debug a thing

### DISCUSSION THREAD

**Discussion by Syilumi** [Bubberstation/Bubberstation#2431] [q&a] [answered]
trying to test an edit i did to vore code but dont really have friends to have clients and enable vore, uncommented the debug define but it didnt seem to help, spawned in humanoids still show the not interested thing anyone know how to make it so i can do the funny eat

also sorry if this isnt the place to do it im just too scared to go onto any discord or contact a coder directly

**Comment by nevimer** [comment, marked as answer]
Try connecting once, then exiting/signing out of byond, and connecting again, you should have yourself and a guest account connected

**Reply by Syilumi** [reply]
YOU CAN DO THAT?! nice

**Reply by Syilumi** [reply]
apart from the guest account having all their keybinds set to unbound it worked
